Services/Masterdata/Samples/RestCalls/masterdata.rest

```diff
@@ -1,5 +1,5 @@
 @ids-url=https://idp/
-@api-url=https://localhost:7029/Masterdata
```

Comment:
How do these `.rest` files work? I have never seen them before! Just curious :).

Reply:
This is something that I've picked from @jstorteboom 
I suppose using Postman is also a good alternative.
For this .rest file, we use the following visual studio code extension: 
  humao.rest-client => https://marketplace.visualstudio.com/items?itemName=humao.rest-client
which interprets the code in this file and allows you to do requests directly in visual studio code without the need to have Postman or other software!